python/problem_1.py

```diff
@@ -0,0 +1,7 @@
+def calculate(limit):
+    result = 0
+    for i in xrange(limit):
```

Comment:
please use python 3

Reply:
Try to reduce the complexity of this algorithm

Reply:
Can you please be more specific, not able to identify the actual issue

Reply:
try to find any other way than using for loop

Reply:
Sorry, I couldn't find any other way.